dist/rpm/openQA.spec

```diff
@@ -288,6 +288,19 @@ upgrading the system if devel:openQA packages are stable and contain updates. It
 is complementary to auto-update which also reboots the system and does updates
 regardless of whether devel:openQA contains updates.
 
+%package munin
+Summary:        Munin scripts
```

Comment:
```suggestion
Summary:        Munin scripts for monitoring openQA
```

Reply:
hm, I still wanted to do that before it's merged... :)